lib/Interpreter/CIFactory.cpp

```diff
@@ -203,6 +203,13 @@ namespace {
       }
 #endif
 
+      // Ignore warnings about redeclaration with/without dllexport & dllimport
+      sArguments.addArgument("-Wno-dll-attribute-on-redeclaration");
+      sArguments.addArgument("-Wno-inconsistent-dllimport");
```

Comment:
The lack of many tests failing is the test...
`extern "C" int printf(const char*, ...);` used in many tests.
`__declspec(dllexport) __cdecl int printf(const char*, ...);` according to the headers.